### PR TITLE
Update env var file loading

### PR DESCRIPTION
**Pull request by stuartleeks**
Update env var file loading to handle quoted JSON (split out from #2634 for easier review)

- Update env file loading scripts to handle quoted JSON
- Combine load_terraform_env.sh into load_env.sh and always import env files with & without TF_VAR prefix

### CODE REVIEW

**Comment by stuartleeks** [comment]
/test

re-run to attempt to resolve https://github.com/microsoft/AzureTRE/actions/runs/3188215607/jobs/5200713265#step:3:7834

File: devops/scripts/load_env.sh [hunk 1]
```diff
@@ -3,16 +3,42 @@ set -o errexit
...
 if [ ! -f "$1" ]; then
   if [ -z "${USE_ENV_VARS_NOT_FILES:-}" ]; then
     echo -e "\e[31m»»» 💥 Unable to find $1 file, please create file and try again!\e[0m"
     #exit
   fi
 else
-  # doesn't work with quotes
-  # shellcheck disable=SC2046
-  export $(grep -v -e '^[[:space:]]*$' -e '^#' "$1" | xargs)
+  # Loop over the relevant lines in the file specified in $1 (passed in after the loop)
```
Comment (tanya-borisova) [inline devops/scripts/load_env.sh:17]: Nice job on leaving these comments 👍

**Review by tanya-borisova — APPROVED** [review]

**Pull request merged**